app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -117,7 +126,53 @@ fun OverviewScreen(
       },
       modifier = Modifier.testTag("overviewScreen"),
       topBar = {
-        TopAppBar(title = { Text(text = "Your trips") }, modifier = Modifier.testTag("topBarTitle"))
+        TopAppBar(
```

Comment:
I liked the colour of this TopAppBar more how it was in Figma.. do you think it looks better like this?

Reply:
I thought it could frame the main screen in a nice manner and highlight it as that. But I see your point and I'll fix it in the UI/UX PR I'm working on this week.